Sources/ArgumentsUtils.swift

```diff
@@ -126,20 +134,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateFound
```

Comment:
Jup, that's what I meant 👍  As the full text would include the error you already have, I'd maybe change it to:

> A template must be chosen either using the named option --template or the path option --templatePath. Default template support has been dropped but you can still access the bundled templates using one of these options.